documentation/directory-structure.md

```diff
@@ -0,0 +1,245 @@
+| Short Link: | [aka.ms/azsdk/spec-dirs](https://aka.ms/azsdk/spec-dirs) |
+|--|--|
+
+<!-- 
+Table of contents generated with VSCode Markdown All in One:
+https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one 
+-->
+
+- [`specification` directory structure](#specification-directory-structure)
+  - [`specification` folder](#specification-folder)
+  - [`resource-manager` and `data-plane` folders](#resource-manager-and-data-plane-folders)
+  - [AutoRest configuration `README.md` files](#autorest-configuration-readmemd-files)
+  - [`stable` and `preview` folders](#stable-and-preview-folders)
+  - [A complete example directory structure of `specification/AzureTeamFolder`](#a-complete-example-directory-structure-of-specificationazureteamfolder)
+  - [Naming guidelines for `specification` folder contents](#naming-guidelines-for-specification-folder-contents)
+  - [`specification/common-types`](#specificationcommon-types)
+  - [Advanced scenario: service group](#advanced-scenario-service-group)
+    - [Service group `common-types`](#service-group-common-types)
+    - [Versioning services in a service group](#versioning-services-in-a-service-group)
+  - [Legacy layouts and OpenAPI specs](#legacy-layouts-and-openapi-specs)
+  - 
+# `specification` directory structure
+
+
+This article describes the directory structure / folder layout of the `specification` folder.
+You may be also interested in following:
+- [Specification index]
+- [Resource Provider list]
+
+> [!IMPORTANT]
+> The layout described in this article is strictly enforced. There exist some exceptions for historical reasons. These exceptions are not allowed going forward.
+
+## `specification` folder
+
+The `specification` folder is the root folder for all service specifications.
+
+Each child of the `specification` folder corresponds to a `service` specification for given Azure team. Here we denote such folder as `AzureTeamFolder`.
+
+Given `AzureTeamFolder` has following structure:
+
+- `AzureTeamFolder/TypeSpecFolder` (multiple folders)
```

Comment:
```suggestion
- `<AzureTeamFolder>/<TypeSpecFolder>` (multiple folders)
```

Perhaps it is just me but I like to use some sort of indicator that these aren't specific values but instead are a placeholder.